ksmt-core/src/main/kotlin/io/ksmt/KContext.kt

```diff
@@ -1347,9 +1682,21 @@ open class KContext(
 
     private val intRemCache = mkAstInterner<KRemIntExpr>()
 
+    /**
+     * Create an Int rem expression.
+     * The result value is either (mod lhs rhs) when rhs >= 0 and (neg (mod lhs rhs)) otherwise.
+     * The result sign matches the [rhs] sign.
+     *
+     * @see mkIntMod
```

Comment:
Probably, because of the error in the `mkIntMod`, it seems wrong as well

Reply:
Fixed in #115